google/cloud/aiplatform/jobs.py

```diff
@@ -1265,7 +1265,12 @@ def from_local_script(
             script_path (str):
                 Required. Local path to training script.
             container_uri (str):
-                Required: Uri of the training container image to use for custom job.
+                Required. Uri of the training container image to use for custom job.
+                Support images in Artifact Registry, Container Registry, or Docker Hub.
+                Vertex AI provides a wide range of executor images with pre-installed
+                packages to meet users' various use cases. See the list of `pre-built containers
+                for training <https://cloud.google.com/vertex-ai/docs/training/pre-built-containers>`.
+                If not using image from this list, please make sure python3 and pip3 are installed in your container.
```

Comment:
If python3 and pip3 aren't installed on a custom container, will the training service or the SDK throw an error? If it's the SDK, could you add some error handling for this case? Otherwise you can leave as is.

Reply:
Currently neither of them will throw the error if python3 and pip3 aren't installed. But `job.run()` will fail and there will be a link to the log, where users can see the reason of failure is `pip not found`. 

If we want to check this error in SDK, we will need to pull and run the docker image inside `from_local_script` method, which will take several minutes or more to finish. And that means the time to initialize a customJob goes from seconds to minutes.

Given that this error is unlikely to happen(customers use SDK means they are python users, so normally the container image they choose to use will have python installed), I think it's not necessary to add this error handling. What do you think?

Reply:
Yes agreed, in that case let's not raise an error in the SDK.